script/client

```diff
@@ -230,11 +223,7 @@ sub handle_result {
 # prepend the API-base if the specified path is relative
 sub prepend_api_base {
     my $path = shift;
-
-    if ($path !~ m/^\//) {
-        $path = join('/', $options{apibase}, $path);
-    }
-    return $path;
+    return join('/', $options{apibase}, $path) if $path !~ m/^\//;
```

Comment:
Now simply returning `$path` when the condition is not true is obfuscated. Maybe better use the ternary operator here?

Reply:
See @perlpunk's comment. It is actually broken (and not only obfuscated).

Reply:
Right. I could use the ternary operator